api/migrations/0013_cityvisitlog_visit_count.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.0.5 on 2018-07-03 04:15
```

Comment:
Remove both the migrations since these are unwanted

Reply:
so you want me to make api call in trip module... Cann you please help me with the implementation?...Tks

Reply:
For a given user, you can get a list of all the city logs.
From those logs you can group them according to cities and have list of visit counts along with the city objects

Reply:
Sir .... I am stuck need help i rewrote the function in view which is 
@api_view(['GET'])
def get_city_visits(request):
    """
    Returns a list of cities visited by a user
    :param request:
    :return: 404 if invalid user not authenticated
    :return: 200 successful
    """
    c = Counter()
    city_visits = CityVisitLog.objects.filter(user=request.user)
    for data in city_visits:
        c[data.city.city_name]+=1


    serializer = CityVisitSerializer(city_visits, many=True)
    return Response(serializer.data)

c is providing desired data(i guess) but i dont know how to pass it through serializer ...Plz help...
I am new to rest framework so plz let me know if i need to learn something for getting this issue resolved

Reply:
You can read this SO thread for a better understanding https://stackoverflow.com/q/19101665/5602759